src/index.js

```diff
@@ -0,0 +1,325 @@
+// Declari
+
+const mysql = require("mysql2");
+const inquirer = require("inquirer");
+require("dotenv").config();
+const figlet = require("figlet");
+const gradient = require("gradient-string");
+const table = require("table");
+const Db = require("./db");
+const { dbOptions } = require("./utils/dbConfig.js");
+
+const config = {
+	// table package style settings
+	border: table.getBorderCharacters("honeywell"),
+};
+
+const {
+	mainQuestions,
+	employeeInfo,
+	departmentInfo,
+	deleteRecord,
+} = require("./utils/questions");
+
+const {
+	employeeQuery,
+	roleQuery,
+	departmentQuery,
+	employeeByManager,
+} = require("./utils/queries");
+
+const {
+	generateRoleChoices,
+	generateEmployeeChoices,
+	generateDepartmentChoices,
+} = require("./utils/utils.js");
+
+const init = async () => {
+	// Welcome message function that uses both figlet and gradient
+	function welcomeMessage() {
+		const welcomeMsg = `SQL  Employee  Management  System`;
+
+		figlet(welcomeMsg, (err, data) => {
+			console.log(gradient.pastel.multiline(data));
```

Comment:
instead of having to use `gradient.something` all through your code, we can use object destructuring when importing the package to pull out just the methods we need. for example `const {instagram, pastel, summer} = require("gradient-string");`